libs/labelbox/src/labelbox/adv_client.py

```diff
@@ -0,0 +1,115 @@
+import io
+import json
+import logging
+from typing import Dict, Any, Optional, List, Callable
+from urllib.parse import urlparse
+
+import requests
+from requests import Session, Response
+
+logger = logging.getLogger(__name__)
+
+
+class AdvClient:
```

Comment:
As `adv` already exposes a public interface (for `advlib`) and it simplifies the integration (by avoiding to proxy requests through our `GQL` API, especially in the case of vector imports) I chose to add an `adv` client.

Some of this code is copied directly from https://github.com/labelbox/advlib.